packages/viewer/src/lib/transforms/hits.ts

```diff
@@ -49,5 +53,7 @@ export function testTriangleMesh(
 
   const r = Vector3.dot(edge2, q) / det;
 
-  return !isNaN(r) && r > epsilon;
+  // Ignore the case where the computed hit position is negative
+  // if in orthographic, as the near plane can be a negative value.
+  return !isNaN(r) && (r > 0 || frame.scene.camera.isOrthographic());
```

Comment:
A better solution would be if `transformPointToOrthographicRay()` constructed a ray where the origin was at the near plane.

Reply:
I actually do think that should be the case, so thinking about this again, I'm not sure my comment here is accurate. I'll update my comment to reflect that it's a bit of a hack to correct the orthographic case where it's moving toward the camera, and create a ticket to revisit this logic

Reply:
https://vertexvis.atlassian.net/browse/PLAT-1549